src/lib/components/collections/collections-list/Folder.svelte

```diff
@@ -276,24 +278,118 @@
       disabled: false,
     },
   ];
+  let requestCount: number = 0;
+  let folderCount: number = 0;
+  let deleteIds: string[] = [];
   $: {
     if (activePath) {
       if (activePath.collectionId === collection.id) {
         visibility = true;
       }
     }
+    if (collection) {
+      deleteIds.length = 0;
+      requestCount = 0;
+      folderCount = 0;
+      collection.items.forEach((item) => {
+        if (item.type === ItemType.FOLDER) {
+          deleteIds.push(item.id);
+          folderCount++;
+          requestCount += item.items.length;
+          for (let i = 0; i < item.items.length; i++) {
+            deleteIds.push(item.items[i].id);
+          }
+        }
+        if (item.type === ItemType.REQUEST) {
```

Comment:
use else if here or it will always go inside this.

Reply:
resolved